### PR TITLE
Update CONCOCT module(s): conda build bump, stub, and nf-test 

### PR DESCRIPTION
**Pull request by jfy133**
For the last three modules (same changes for all)

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/concoct/concoct/tests/main.nf.test.snap [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+    "sarscov2 - bam - stub": {
```
Comment (SPPearce) [inline modules/nf-core/concoct/concoct/tests/main.nf.test.snap:2]: This looks strange, with no md5sums

File: modules/nf-core/concoct/extractfastabins/main.nf [hunk 2]
```diff
@@ -40,4 +40,17 @@ process CONCOCT_EXTRACTFASTABINS {
         concoct: \$(echo \$(concoct --version 2>&1) | sed 's/concoct //g' )
     END_VERSIONS
     """
+
+    stub:
+    def args   = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    mkdir -p ${prefix}
+    touch ${prefix}/${prefix}.fa.gz
```
Comment (SPPearce) [inline modules/nf-core/concoct/extractfastabins/main.nf:49]: nf-test doesn't like gz files that aren't actually zipped.
```suggestion
    echo "" | gzip > ${prefix}/${prefix}.fa.gz
```

File: modules/nf-core/concoct/extractfastabins/tests/main.nf.test.snap [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{
+    "sarscov2 - bam - stub": {
```
Comment (SPPearce) [inline modules/nf-core/concoct/extractfastabins/tests/main.nf.test.snap:2]: Again, this has no output?

File: modules/nf-core/concoct/extractfastabins/tests/nextflow.config
Comment (SPPearce) [inline modules/nf-core/concoct/extractfastabins/tests/nextflow.config:1]: Do we need this?

**Comment by jfy133** [comment]
@SPPearce Re the empty stub files, suprisingly this seems to be a silent fail. I have some channel manipulation during setup which I guess is failing for some reason, and causes no input to actually get to the module under testing...  not sure why this works during the actual test and not the stup...

File: modules/nf-core/concoct/extractfastabins/tests/nextflow.config
Comment (jfy133) [reply modules/nf-core/concoct/extractfastabins/tests/nextflow.config:1]: Apparnetly not, removed

File: modules/nf-core/concoct/concoct/tests/main.nf.test.snap (hunk 1 above)
Comment (jfy133) [reply modules/nf-core/concoct/concoct/tests/main.nf.test.snap:2]: Found it, everything shoud have md5sums now

File: modules/nf-core/concoct/extractfastabins/tests/main.nf.test.snap (hunk 3 above)
Comment (jfy133) [reply modules/nf-core/concoct/extractfastabins/tests/main.nf.test.snap:2]: See above

**Comment by SPPearce** [comment]
Does one of the upstream tools not have a stub? Or the stub doesn't produce the output file?

**Comment by jfy133** [comment]
> Does one of the upstream tools not have a stub? Or the stub doesn't produce the output file?

All the tools have the stub. All I can think was the upstream stub was also broken... (in that it also produced no files...) I don't know.

REgardless changed in the input system and all works now, should be ready for re-review!

**Review by SPPearce — APPROVED** [review]

**Pull request merged**